src/pages/RepoPage/PullsTab/PullsTab.tsx

```diff
@@ -109,7 +109,7 @@ function PullsTab() {
       <div className="flex flex-row gap-3">
         <div className="flex items-center justify-center gap-3">
           <label className="text-sm font-semibold">View:</label>
-          <div>
+          <div className="w-32">
```

Comment:
Making this change after talking to Adalene: the current width of the multiselect changes with what is selected and wraps "Clear selected" so we are making it consistent